queries.sql

```diff
@@ -122,4 +122,60 @@ INNER JOIN owners ON owner_id = owners.id
 GROUP BY owners.full_name) AS animals_per_owner 
 WHERE animals_count = (SELECT max(animals_count) FROM (SELECT full_name, COUNT(animals.name) as animals_count FROM animals
 INNER JOIN owners ON owner_id = owners.id
-GROUP BY owners.full_name) AS animals_per_owner);
\ No newline at end of file
+GROUP BY owners.full_name) AS animals_per_owner);
+
+---------------------------------------------------------------------------------------
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name, visits.date_of_visit FROM animals 
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id 
+WHERE vets.id = 1 
+ORDER BY date_of_visit DESC LIMIT 1;
```

Comment:
- [ ] Please refrain from using the `id` in conditions. Instead, you can use the name or any other property that this record has. 
- The reason is that in a real database, it'll be near impossible to keep up with every record's `id` and so you'll use the properties that we can memorize and distinguish.